### DISCUSSION TITLE
Master branch merge timeline?

### DISCUSSION THREAD

**Discussion by andyphenix** [kartoza/docker-geoserver#500] [General]
Hi there, we (ESA / Sitkatech) maintain a downstream fork & try to build off the master branch. Issue #486 is blocking us at the minute. Just wondering if you had a timeline as to if / when you expect to merge develop --> master? Thanks!

**Comment by NyakudyaA** [comment]
@andyphenix That has been fixed. Can you see the latest develop branch and build against that? All tags published in https://github.com/kartoza/docker-geoserver/issues/496 are working fine with the gdal provider

**Comment by andyphenix** [comment]
Yes, was able to do that. However we build 'locally' and use the master branch for that. Is it your intention to merge develop to master at some point? Or do you now consider your develop branch to be the 'release' branch? (No worries if so, just trying to understand your branching strategy so we can adjust locally).

**Comment by NyakudyaA** [comment]
Currently, the develop is the release branch. I would have loved to be able to push all changes to master once it's tested but for now, they are no unit tests and develop and master would not have a difference. So I think we need to add tests as priority
